validator/testcases/regex.py

```diff
@@ -1546,6 +1547,23 @@ def tests(self):
             compat_type="error")
 
 
+@register_generator
+class Gecko37RegexTests(CompatRegexTestHelper):
+    """Regex tests for Gecko 37 updates."""
+
+    VERSION = FX37_DEFINITION
+
+    def tests(self):
+        yield self.get_test(
+            r"\bnsIDownloadManagerUI\b",
```

Comment:
This is just a regex test so if this appears anywhere in the code then the warning/error will be registered. This unfortunately includes comments. It might be nice to run the code through something that strips the comments out first.